ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__metal-4.19-nightly-x86.yaml

```diff
@@ -119,6 +119,27 @@ tests:
     - chain: openshift-qe-data-path-tests
     workflow: openshift-qe-installer-bm-deploy
   timeout: 7h55m0s
+- always_run: false
+  as: data-path-6nodes-external
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    cluster_profile: metal-perscale-cpt
```

Comment:
Please use the metal-perfscale-jetlag during development, we can transition to the metal-perfscale-cpt profile when the test is ready